packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFDataConverters.m

```diff
@@ -176,6 +176,18 @@ WKAudiovisualMediaTypes FWFWKAudiovisualMediaTypeFromEnumData(
       allHttpHeaderFields:request.allHTTPHeaderFields ? request.allHTTPHeaderFields : @{}];
 }
 
+FWFWKNavigationResponseData *FWFWKNavigationResponseDataFromNavigationResponse(
+    WKNavigationResponse *response) {
+  return [FWFWKNavigationResponseData
+      makeWithResponse:FWFNSHttpUrlResponseDataFromNSURLResponse(response.response)
+          forMainFrame:@(response.forMainFrame)];
+}
+
+FWFNSHttpUrlResponseData *FWFNSHttpUrlResponseDataFromNSURLResponse(NSURLResponse *response) {
+  NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *)response;
```

Comment:
Can you add a note above about why this is casting an `NSURLResponse` to a `NSHTTPURLResponse`? I read in the docs that this is expected to always work and I'm assuming this is the reason this was done.

e.g: https://developer.apple.com/documentation/foundation/nsurlresponse#overview